app/assets/stylesheets/donor.css.scss

```diff
@@ -236,3 +236,12 @@
     font-weight: 500;
   }
 }
+
+.best_in_place:hover {
```

Comment:
idk how this will work... but is it possible to fix it from spazzing the fuck out when you click it?  there's a green bar above and below it and it freaks out when you move the mouse around

![screen shot 2015-04-07 at 23 17 52](https://cloud.githubusercontent.com/assets/2336036/7039890/5e5dc6c4-dd7c-11e4-96a1-29a32ffe1914.png)